lib/bloc/menubloc/menu_bloc.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:bloc/bloc.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:nutrilab/bloc/menubloc/menu_event.dart';
+import 'package:nutrilab/bloc/menubloc/menu_state.dart';
+import 'package:nutrilab/models/menuitemmodel.dart';
+
+class MenuBloc extends Bloc<MenuEvent, MenuState> {
+  final String userId;
+
+  MenuBloc(this.userId) : super(MenuInitial()) {
+    on<LoadMenuItems>(_onLoadMenuItems);
```

Comment:
I think you can start loading data right in constructor, no need to send event to start.

Reply:
Can I emit loading state without an event

Reply:
Yes, you can emit any state at any moment regardless of events. Events are just triggers for some logic, here we don't need any trigger from UI to start loading. 
You can just add 
`add(LoadMenuItems())` after this line